### DISCUSSION TITLE
How to disable react-native-reanimated 2 ?

### DISCUSSION THREAD

**Discussion by holyris** [software-mansion/react-native-reanimated#3600] [Q&A]
I needed to install this library to use @react-navigation/drawer, but it seems that react-native-reanimated@2 prevent me from using remote debugging. 
From the react-navigation drawer documention, it is specified that it's possible to use reanimated 1 instead of 2 with the prop [useLegacyImplementation ](https://reactnavigation.org/docs/drawer-navigator#uselegacyimplementation). 
It's also told that "**The Drawer Navigator supports both Reanimated 1 and Reanimated 2. If you want to use Reanimated 2, make sure to configure it following the [installation guide](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation).**". 
Should I understand that it's possible to not use reanimated 2 by omitting the config described [here](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation/)? I tried but end up with issues like [this](https://github.com/software-mansion/react-native-reanimated/issues/1875).

Not having a debugger is awful, please help :(

**Comment by tomekzaw** [comment]
> it seems that react-native-reanimated@2 prevent me from using remote debugging.

Hey, you can use remote debugger (such as Chrome DevTools or Hermes Debugger in Flipper) with Reanimated 2.

For details, see https://docs.swmansion.com/react-native-reanimated/docs/next/guide/debugging#summary
